packages/create/src/cli.ts

```diff
@@ -15,7 +17,7 @@ program
   .version(version)
   .argument('[project-name]', 'Name of the project to create')
   .option('-t, --template <template>', 'Template to use (webpack, vite)')
-  .action(async (projectName: string = '.', options: { template?: string }) => {
+  .action(async (targetDir: string = '.', options: { template?: string }) => {
```

Comment:
Defaulting immediately to the current directory could be a bit confusing as there's no indication that this is happening until creation has already started.

I feel like this project name could be another prompt just like the template. That way if a name isn't provided to the CLI you can indicate that either a directory name or `.` is valid.